Automattic/simplenote-android
# Update List Layout

~These changes are pending the [***Update Pinned Icon*** pull request](https://github.com/Automattic/simplenote-android/pull/823) and should only be merged subsequently.~

### Fix
Update the note list item layout by reducing the extra line spacing of the note content text from 4dp to 2dp and removing the layout constraint of the note content to be left/start of the note status icons (i.e. ***Pinned*** and ***Published***).  See the screenshots below for illustration.

![update_list_layout](https://user-images.githubusercontent.com/3827611/66154163-1df90100-e5da-11e9-90d0-71d6948177a1.png)

### Test
1. Go to note list.
2. Notice layout changes as mentioned and shown above.

### Review
 Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
